src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="container-fluid">
+  <div class="row mx-0">
+    <cd-card cardTitle="Details"
+             i18n-cardTitle
+             class="col px-3">
+             Text
+    </cd-card>
+    <cd-card cardTitle="Status"
+             i18n-cardTitle
+             class="col-6 px-3">
+             Text
+    </cd-card>
+
+    <cd-card cardTitle="Capacity"
+             i18n-cardTitle
+             class="col px-3">
+             Text
+    </cd-card>
+  </div>
+
+  <div class="row mx-0">
+    <cd-card cardTitle="Inventory"
+             i18n-cardTitle
+             class="col px-3">
+             Text
+    </cd-card>
+
+    <cd-card cardTitle="Capacity utilization"
+             i18n-cardTitle
+             class="col-6 px-3">
+             Text
+    </cd-card>
+
+    <cd-card cardTitle="Events"
+             i18n-cardTitle
+             class="col px-3">
+             Text
+    </cd-card>
+  </div>
+</div>
```

Comment:
Hey @bryanmontalvan, some pointers that I think we should address here.

- I think we should try to abstract components, keeping them simple and reusable, so we should probably look out for other way to include our grid. Doing so will make it easier to make any changes in the future without over complicating our components. With this in mind I would make another component to tackle the grid system. Also, I would prefer to have class columns in different `<div>` tags, so we could have rows and columns separated from the cards logic.

- `class="col"` could cause some issues with our layout since it will resize accordingly to it's content. Meaning the following, if the content of the card is bigger than the card it will resize it's width to keep it insidife of it, like so,
 
Example:
![Screenshot from 2022-07-17 10-53-14](https://user-images.githubusercontent.com/106810838/179415482-371370ab-b821-4a37-86f6-9f390b72423a.png)


  And what we want here is to keep the three columns layout, this is why I would go for something like:
  
  `class="col-sm-3"`  for the small columns and
  `class="col-sm-6"`  for the bigger columns
  
  This way we already assign the size of the column and it won't change until the screen size is small, this is where the `sm` class comes in handy for the smaller screens and it will break our layout to keep everything visible,
  
Example:
![Screenshot from 2022-07-17 11-21-02 (1)](https://user-images.githubusercontent.com/106810838/179417691-255322b2-9fc2-485c-b632-2839b74dddea.png)

In this example the first row includes the `sm` class while the second doesn't.

As always, let me know what you think. And @nizamial09 you too as well, if you could check it out. Thank you!